# Написать функцию которая принимает на вход число от 1 до 100.
# Если число равно 13, функция поднимает исключительную ситуации ValueError
# иначе возвращает введенное число, возведенное в квадрат.
#
# Далее написать основной код программы.
# Пользователь вводит число.
# Введенное число передаем параметром в написанную функцию и печатаем результат,
# который вернула функция. Обработать возможность возникновения исключительной ситуации,
# которая поднимается внутри функции.


def is_thirteen(num):
    if num == 13:
        raise ValueError
    else:
        return num ** 2


if __name__ == '__main__':
    while True:
        number = input('Введите число, для выхода введите \'exit\': ')
        if number != 'exit':
            try:
                number = int(number)
            except ValueError:
                print('Вы ввели не число, для выхода введите \'exit\'')
                continue
        else:
            break
        try:
            print(is_thirteen(number))
        except ValueError:
            print('Вы ввели 13 - это недопустимо, для выхода введите \'exit\'')
